chingu-coders/Voyage2-Turtles-02
# update with settings

## Purpose
* Add settings feature
* Clean up JS code
* Add more functionality

## Approach/Fixes
* Created `clear` buttons for validators and page speed
* Add JS file for settings
* Update css to make features more consistently styled
* Removed majority of ESLint errors from JS code

## Testing
* Pull this into your local maching
* Check that `clear` buttons work
* Check that html/css validators work
* Check that the settings container pops up on `click`
* Navigate through `about` and `contribute` tabs in settings to see that they change

## NOTES
* Quicklinks was buggy when using `fadeIn()` and `fadeOut()` so right now it's just `show()` and `hide()`
